lua/keymappings.lua

```diff
@@ -1,24 +1,25 @@
 local M = {}
 local Log = require "core.log"
 
-local generic_opts_any = { noremap = true, silent = true }
-
-local generic_opts = {
-  insert_mode = generic_opts_any,
-  normal_mode = generic_opts_any,
-  visual_mode = generic_opts_any,
-  visual_block_mode = generic_opts_any,
-  command_mode = generic_opts_any,
+local generic_opts = { noremap = true, silent = true }
+local opts = {
+  insert_mode = generic_opts,
+  normal_mode = generic_opts,
+  visual_mode = generic_opts,
+  visual_block_mode = generic_opts,
+  command_mode = generic_opts,
+  operator_mode = generic_opts,
   term_mode = { silent = true },
 }
 
 local mode_adapters = {
   insert_mode = "i",
   normal_mode = "n",
   term_mode = "t",
-  visual_mode = "v",
-  visual_block_mode = "x",
+  visual_and_select_mode = "v",
```

Comment:
Why renaming to visual_and_select_mode ? I would prefer that we keep the came terminology as vim.
Also, select mode is a different mode, see `:help vim-modes`

Reply:
From the vim docs:

![2021-09-16_10-03](https://user-images.githubusercontent.com/65440103/133550136-821d3505-cea1-400d-aa3c-ad0ac70e496d.png)

Reply:
Alright, I think 's' is missing, do you mind adding it ?
I would also prefer `visual_and_select_mode` -> `visual_select_mode`